admittance_node:
  ros__parameters:
    equilibrium.position: [0.491834, 0.13328, 0.48792]
    equilibrium.orientation: [7.2002e-05, 0.707179, -7.20168e-05, -0.707035]
